src/api/routes/articleRouter.js

```diff
@@ -87,5 +88,9 @@ articleRouter.post('/:slug/share/gmail',
   shareArticle,
   articleController.socialShareArticle);
 
+articleRouter.post('/:slug/:commentId/report',
```

Comment:
I don't see any validations here @MCFrank16 
Yet it's a POST

Reply:
I had to validate nothing as you just hit the endpoint and the magic happens thereafter

Reply:
Alright @MCFrank16